datalab/test_datalab.py

```diff
@@ -19,14 +19,11 @@ def verify_instance(self, name):
 
     @parameterized.expand(
         [
-            ("STANDARD", "1.1", ["m"], "python2"),
-            ("STANDARD", "1.1", ["m"], "python3"),
             ("STANDARD", "1.2", ["m"], "python2"),
             ("STANDARD", "1.2", ["m"], "python3"),
             ("STANDARD", "1.3", ["m"], "python2"),
             ("STANDARD", "1.3", ["m"], "python3"),
-            ("STANDARD", "1.4", ["m"], "python2"),
-            ("STANDARD", "1.4", ["m"], "python3"),
+            ("STANDARD", "1.4", ["m"], "python3-default"),
```

Comment:
What is python3-default? why do we want to use it?

Reply:
It means that we don't need to install Python 3 in 1.4 image because it's install by default (see test code below)